pkg/core/model/message.proto

```diff
@@ -0,0 +1,22 @@
+syntax = "proto3";
+package model;
+import "google/protobuf/any.proto";
+
+message MessageHeader {
+    string ID = 1;
+    string ParentID = 2;
+    int64 Timestamp = 3;
+    bool Sync = 4;
+}
+message MessageRoute {
+    string Source = 1;
+    string Group = 2;
+    string Operation = 3;
+    google.protobuf.Any Resource = 4;
```

Comment:
let us change resource into type struct to resolve the resource hardcode problem

Reply:
@kadisi , can you please check this ?

Reply:
@sids-b @rohitsardesai83  may be one single type struct dose not solve resource hardcode problem.  
we need different type struct. because currently resource has several formats, as follows:

```
controller:
/node/{node_id}/{namespace}/{resource type}/ 
/node/{node_id}/{namespace}/{resource type}/{resource id}

devicecontroller:
/node/{node_id}/{resource type}/
/node/{node_id}/{resource type}/{resource id}

metamanager:
/{namespace}/{resource type}/
/{namespace}/{resource type}/{resource id}


eventbus:
SYS/dis/upload_records
topic:
$hw/events/upload/
$hw/events/device/+/state/update"
$hw/events/device/+/twin/+"
$hw/events/node/+/membership/get"

```

so, `Any` Type is usefull to support different type struct.

Reply:
But is this case also we may still have the resource hardcode or nil pointer exception problem right ?